File: LeetCode/cpp/973.cpp
class Solution {
public:
    vector<vector<int>> kClosest(vector<vector<int>> &points, int K) {
        vector<vector<int>> ret;
        sort(points.begin(), points.end(), [](vector<int> &v1, vector<int> v2) {
            return (pow(v1[0], 2) + pow(v1[1], 2)) < (pow(v2[0], 2) + pow(v2[1], 2));
        });
        for (int i = 0; i < K; ++i)
            ret.push_back(points[i]);
        return ret;
    }
};

File: LeetCode/python3/973.py
class Solution:
    def kClosest(self, points, K):
        """
        :type points: List[List[int]]
        :type K: int
        :rtype: List[List[int]]
        """
        points.sort(key=lambda p: pow(p[0], 2) + pow(p[1], 2))
        return points[:K]
